game/templates/game/board.html

```
@@ -443,6 +464,10 @@
                 </div>
             </div>
 
+           <div style="display: flex; flex-direction: column; gap: 12px;">
+    <button class="btn btn-gold" id="gameOverStartBtn" style="padding: 12px; font-size: 1rem;">Start New Game</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)